Automattic/vip-go-mu-plugins
# Search: Remove hook to automatically create index for new blogs on multisites

## Description
We want to allow the constant `EP_IS_NETWORK` to be enabled (for cross-blog searching) without automatically creating indexes everytime a subsite is created on a multisite.

## Changelog Description

### Plugin Updated: Enterprise Search

Remove hook to automatically create index for new blogs.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
0) Check out PR
1) Create multisite with ES and `EP_IS_NETWORK` defined
2) Create new subsite
3) Run `wp vip-search get-indexes`
4) Ensure index has not been created for it.
